core/arch/arm/mm/core_mmu.c

```diff
@@ -76,6 +82,16 @@ static bool pbuf_inside_map_area(unsigned long p, size_t l,
 	return core_is_buffer_inside(p, l, map->pa, map->size);
 }
 
+static struct map_area *find_map_by_type(enum teecore_memtypes type)
+{
+	struct map_area *map;
+
+	for (map = static_memory_map; map->type != MEM_AREA_NOTYPE; map++)
```

Comment:
This is how we test below in `find_map_by_{va,pa}()`. The array is supposed to be terminated with `MEM_AREA_NOTYPE`.